ComputeClient/Compute.Contracts/Snapshot/Snapshot.cs

```diff
@@ -2437,18 +2451,14 @@ public bool replica
         }
     }
 
+
     /// <remarks/>
-    [System.Xml.Serialization.XmlAttributeAttribute()]
+    [Obsolete("archived is deprecated, please use archiveStatus instead.")]
```

Comment:
Have marked this method as deprecated so to encourage the use of the archiveStatus method longer term as convience methods like this are not idea for XSD generated classes.